pkg/tests/mappings/mappings_test.bzl

```diff
@@ -27,7 +27,12 @@ load(
     "strip_prefix",
     "REMOVE_BASE_DIRECTORY",
 )
-load("//tests/util:defs.bzl", "directory", "generic_base_case_test", "generic_negative_test")
+load("//tests/util:defs.bzl",
+     "directory",
+     "fake_artifact",
+     "generic_base_case_test",
+     "generic_negative_test"
+)
 
```

Comment:
(Github won't let me add a comment in the relevant location)

On lines 51-52/56-57, there's a comment that mentions `.exe` files and `sh_binary`.  This comment should be updated to remove the mention of `sh_binary` if it's still needed, or the logic immediately below should be removed if possible.

Reply:
Done.